scst-store/tls-configuration.hbs.md

```diff
@@ -0,0 +1,53 @@
+
+# Setting up TLS in Metadata Store
+
+###  <a id="Setting up tls namespace and secretName"></a>Setting up tls namespace and secretName
+
+In `tap-values` file you can set namespace and secretName as shown below.
+
+---
+```yaml
+tls:
+  namespace: "The targeted namespace for secret consumption by the HTTPProxy."
+  secretName: "The name of secret for consumption by the HTTPProxy."
+```
+---
+In `tap-values` file you also set Minimum TLS version supported by metadata store and ciphers supported.
+
+---
+```yaml
+tls.server.minTLSVersion:
+  type: string
+  description: 'Minimum TLS version supported. Value must match version names from https://golang.org/pkg/crypto/tls/#pkg-constants. (default "VersionTLS12")'
+  default: "VersionTLS12"
+tls.server.rfcCiphers:
+  type: array
+  description: 'List of cipher suites for the server. Values are from tls package constants (https://golang.org/pkg/crypto/tls/#pkg-constants). If omitted, the default Go cipher suites will be used'
+  default: 
+    - TLS_AES_128_GCM_SHA256
+    - TLS_AES_256_GCM_SHA384
+    - TLS_ECDHE_ECDSA_WITH_AES_128_GCM_SHA256
+    - TLS_ECDHE_ECDSA_WITH_AES_256_GCM_SHA384
+    - TLS_ECDHE_RSA_WITH_AES_128_GCM_SHA256
+    - TLS_ECDHE_RSA_WITH_AES_256_GCM_SHA384
+```
```

Comment:
This is not representative of what a user would put into their tap-values.yaml file, which is misleading from the sentence above. It would be better if this were representative of valid values for a user to input.